client/dashboard/me/notifications-emails/pause-all-emails/confirmation-modal.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+	Modal,
+	Button,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { ButtonStack } from '../../../components/button-stack';
+
+interface Props {
+	onCancel: () => void;
+	onConfirm: () => void;
+}
+
+export const ConfirmationModal = ( { onCancel, onConfirm }: Props ) => {
```

Comment:
I think we can use this newly created component instead: https://github.com/Automattic/wp-calypso/blob/77750a168b6326ed681e13d936fcb5cb1ac5c085/client/dashboard/components/confirm-modal/index.tsx